tests/subsys/llext/simple/src/export_dependent_ext.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Test importing symbols, exported by other LLEXTs */
+
+#include <zephyr/ztest.h>
+#include <zephyr/llext/symbol.h>
+#include <zephyr/sys/util.h>
+extern int test_dependency(short a, short b);
+
+void test_dependent(void)
```

Comment:
Nit: I think this should be `test_entry`, for consistency with all other test suites.